src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/src/images/logo.png"
+        class="logo_link"
+      >
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="link apple">
+            <a
+              href="#"
+              class="first_link nav_link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="link samsung">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="link smartphones">
+            <a
+              href="#"
+              class="nav_link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="link">
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="link gadgets">
```

Comment:
you don't those styles
```suggestion
          <li class="link">
```
just use the link